vendor/scripts/register-game-libraries.js

```diff
@@ -0,0 +1,3 @@
+require.register("game-libraries", function(exports, require, module) {
+	exports = {createjs: window.createjs};
```

Comment:
This adds it to `require.list()`  thought thats not actually needed.  Ill try removing the override in the ModuleLoader.